config/default.conf

```diff
@@ -30,7 +30,7 @@ url_repository = ""
 url_transformation = ""
 url_registration = ""
 
-redirect_to_website = "http://www.copyrighthub.org/"
+redirect_to_website = "https://openpermissions.org"
```

Comment:
This needs to stay as copyrighthub (or be updated in local.conf in provisioning) otherwise our resolution redirect of hubkeys will stop working